packages/preview/sdu-typst-touying/0.1.0/example/main.typ

```diff
@@ -0,0 +1,402 @@
+#set text(font: "Source Han Serif", lang: "CN",region: "CN")
+#import "../lib.typ": *
+// #import "test-theme.typ": *
+// #show: "调试信息：模板加载成功111"
+// #show: dewdrop-theme.with(aspect-ratio: "16-9")
```

Comment:
It would be cleaner to remove these lines I think, or add a comment to explain why they can be useful, so that users of your template know why they are here.

More generally there are a lot of commented lines in your template. I would either remove them entirely, uncomment them, or at least provide a bit more context to tell what they would achieve if uncommented if the goal is to showcase other useful Typst packages to newcomers.